backend/logic/courses/courses.js

```diff
@@ -0,0 +1,10 @@
+const mysql = require('../../sql/connection');
+
+async function queryCourseByTag(tag){
```

Comment:
Forgot to mention, verify that tag is not undefined

Reply:
done!